infra/base-images/base-builder-go-codeintelligencetesting/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-go
+
+# /root/.go-CodeIntelligenceTesting/bin is for the standard Go binaries (i.e. go, gofmt, etc).
+# $GOPATH/bin is for the binaries from the dependencies installed via "go get".
+ENV PATH /root/.go-CodeIntelligenceTesting/bin:$PATH
```

Comment:
It would be better to just replace everything in /root/.go/ instead of adding this one. Is it possible at all to build the patched Go in a /tmp directory and move it back to /root/.go/ at the end?

Reply:
Yep, fixed in [this commit](https://github.com/google/oss-fuzz/commit/057e2f121b9d2675359d10a729b5397484fc8b90).